script/AdaptersDeployer.s.sol

```diff
@@ -61,6 +61,8 @@ contract AdaptersActionBatcher is CommonActionBatcher {
             report.layerZeroAdapter.rely(address(report.common.protocolGuardian));
             // Temporary ward for initial wire, self-denies after
```

Comment:
`self-denies after` should be removed now

Reply:
[5e738d8](https://github.com/centrifuge/protocol/pull/654/commits/5e738d860b4d9fd3b443c735a156f8918db676bd)